grails-app/views/project/espOverview.gsp

```diff
@@ -444,7 +441,7 @@
         if (simplifiedReportingViewModel.canViewSubmissionReport() && adminActivity) {
 
             var $mySites = $('#sites-tab');
-            $mySites.on('shown.bs.tab', function() {
```

Comment:
I am assuming this was the result of a bad merge, but will create a regression bug.